config/elastic-apm-laravel.php

```diff
@@ -63,5 +63,8 @@
             // If a query takes longer then 200ms, we enable the query log. Make sure you set enabled = 'auto'
             'threshold' => env('APM_THRESHOLD', 200),
         ],
+
+        // Render source code in stack traces
+        'renderSource' => env('APM_RENDERSOURCE', true),
```

Comment:
Can you move this into the `span` configuration block?

Reply:
it already is in `spans`array ;)

Reply:
Oh I see 😅